R/meetverdeling.R

```diff
@@ -61,7 +68,7 @@ meetverdeling <- function(d, jaren_overlap = 0) {
     res <- data.frame(tijdsperiode = c("1e planperiode KRW (2004-2009)",
                                        "2e planperiode KRW (2010-2015)",
                                        "3e planperiode KRW (2016-2024)"),
-                      voldoende_metingen = c(all(min_p1$meetjaar %in% periode1),
+                      voldoende.metingen = c(all(min_p1$meetjaar %in% periode1),
```

Comment:
In handleiding_trendanalyse refereer je naar voldoende_metingen (underscore ipv punt), regel 367. Daar krijg ik nu een error als de vignettes build.

Reply:
Heeft even geduurt voordat ik hiermee weer aan de slag kon. Ik weet niet meer waarom ik die had aangepast. Nu terug naar _ en ook in de beschrijving daar boven de . naar _ aangepast. Zou nu goed moeten gaan denk ik